save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/OrganizationController.kt

```diff
@@ -18,37 +23,47 @@ import java.time.LocalDateTime
 @RequestMapping("/api/organization")
 internal class OrganizationController(private val organizationService: OrganizationService) {
     /**
-     * @param name
+     * @param organizationName
      * @return Organization
      */
-    @GetMapping("/get/organization-name")
-    fun getOrganizationByName(@RequestParam name: String): Organization =
-            organizationService.findByName(name) ?: throw NoSuchElementException("Organization with name [$name] was not found.")
+    @GetMapping("/{organizationName}")
+    @PreAuthorize("permitAll()")
+    fun getOrganizationByName(@PathVariable organizationName: String) = Mono.fromCallable {
+        organizationService.findByName(organizationName)
+    }.switchIfEmpty {
+        Mono.error(NoSuchElementException("Organization with name [$organizationName] was not found."))
+    }
 
     /**
      * @param authentication an [Authentication] representing an authenticated request
      * @return list of organization by owner id
      */
     @GetMapping("/get/list")
-    fun getOrganizationsByOwnerId(authentication: Authentication): List<Organization> {
+    @PreAuthorize("permitAll()")
+    fun getOrganizationsByOwnerId(authentication: Authentication?): Flux<Organization> {
```

Comment:
Why do we need here and in some other places to wrap with `Flux/Mono`, while it worked before without it?